### PR TITLE
Feat/282761  show missing info

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/views/cdo/view/dog-list-panel.njk [hunk 1]
```diff
@@ -28,8 +28,16 @@
...
+              <td class="govuk-!-padding-top-2 govuk-!-padding-bottom-0">{{ dog.name }}</td>
+            {% else %}
+              <td class="govuk-!-padding-top-2 govuk-!-padding-bottom-0 defra-secondary-text">Not entered</td>
+            {% endif %}
+            {% if dog.microchipNumber %}
             <td class="govuk-table__header--numeric govuk-!-padding-top-2 govuk-!-padding-bottom-0">{{ dog.microchipNumber }}</td>
+            {% else %}
+            <td class="govuk-table__header--numeric govuk-!-padding-top-2 govuk-!-padding-bottom-0 defra-secondary-text">Not entered</td>
+            {% endif %}
           </tr>
```
Comment (jbarnsley10) [inline app/views/cdo/view/dog-list-panel.njk:41]: Do we not display microchipNumebr2 here, even when entered? Perhaps an oversight

Comment (whitewaterdesign) [reply app/views/cdo/view/dog-list-panel.njk:41]: good point, will add

Comment (whitewaterdesign) [reply app/views/cdo/view/dog-list-panel.njk:41]: how do you feel about us adding in a subsequent ticket?

**Review by jbarnsley10 — APPROVED** [review]
LGTM

**Pull request merged**